tests/test_filters.py

```diff
@@ -690,3 +772,96 @@ def test_flate_decode__not_rectangular(caplog):
     expected = get_data_from_url(url, name=name)
     assert actual_image.getvalue() == expected
     assert caplog.messages == ["Image data is not rectangular. Adding padding."]
+
+
+def test_main_decode_brotli_without_brotli_installed_subprocess(tmp_path):
```

Comment:
We only need to test the missing installation once, not multiple times. Thus we can avoid using subprocesses everywhere.